File: src/main/java/com/hyak4j/guess/Game.java
package com.hyak4j.guess;

import java.util.Random;
import java.util.Scanner;

public class Game {
    public static void main(String[] args) {
        Random random = new Random();
        int secret = random.nextInt(10) + 1;  // 1~10

        Scanner scanner = new Scanner(System.in);
        int number = 0;
        while (number != secret){
            System.out.println("Please enter a number: ");
            number = scanner.nextInt();
            System.out.println(number);
            if (number < secret){
                System.out.println("Higher");
            }else if(number > secret){
                System.out.println("Lower");
            }else {
                System.out.println("Great, the number is " + secret);
            }
        }
/*
        int i = 0;
        do{
            System.out.println(i);
            i ++;
        }while (i < 3);
*/

    }
}

File: src/main/kotlin/com/hyak4j/kotlin/Game.kt
package com.hyak4j.kotlin

import java.util.*

fun main() {
//    val scanner = Scanner(System.`in`)
    val secret = Random().nextInt(10) + 1
    var number = 0

    while (number != secret){
        println("Please enter a number: ")
//        number = scanner.nextInt()
        number = readLine()!!.toInt()
        if (number > secret){
            println("Lower~")
        }else if (number < secret){
            println("Higher~")
        }else{
            println("Bingo! the number is $number")
        }
    }
}